demo/03-create-cluster.sh

```diff
@@ -99,6 +99,39 @@ IDENTITY_UAMIS_JSON_MAP=$(echo -n "${IDENTITY_UAMIS_JSON_MAP}" | jq \
 
 }
```

Comment:
The changes of the 03 create cluster script should be moved to its own dedicated separated MR.

The reason is because as soon as this is merged etcd with pmk will be required, but it will take like approx 1-1.5 hours for the changes to be rolled out to dev. We should only merge the demo 03 script once we know the changes have been rolled out to shared dev to minimize the time the scripts won't work